codc/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .codc import *
```

Comment:
The choice to move from a single-file python module, to a package that imports everything into its __init__.py from a single file seems ... odd ... to me.

What is the motivation for this? (I am persuadable, not having any particular knowledge/expertise on python packaging).

Reply:
@simonsmart, I moved this to a package so that I could use the ``package_data`` command in ``setup.py`` to include the ``processed_odc.h`` file in the distribution. When downloading directly from PyPI using pip, the ``processed_odc.h`` file is not included rendering the codc module unusable.

I'm still learning about python packaging and distribution (not a simple subject!). I tried several approaches to including the ``processed_odc.h`` in the distribution while keeping the codc.py as a standalone module, but nothing worked. ``setup.py`` however, makes it easy to include extra files if you structure your code as a package (instead of a standalone module).

The python package requires an ``__init__.py`` file, so it would also work to simply rename codc.py to ``__init__.py`` if that seem preferable to you. I don't have a strong preference one way or the other - I was just mimicking how the pyodc package was structured.